frontend/src/helpers/messages.js

```diff
@@ -9,4 +9,5 @@ export default {
   mapViewerName: 'Map Viewer',
   gemBrowserName: 'GEM Browser',
   interPartName: 'Interaction Partners',
+  fileSaveError: 'There was a problem saving the file. Please try using a different browser or contact us.',
```

Comment:
Having seen this message and wanting to contact us, a user should not need to spend any time searching for the contact information. Some suggestions are to: include a link to the contact section in About, or include the email address.

Reply:
This is based on the `unknownError` in `messages.js`. I agree that it is better to make it a email link, do you want this to be changed for all "contact us" messages in this PR?

Reply:
I'd prefer it this would be done for all messages, but it's up to you.